.github/PULL_REQUEST_TEMPLATE

```diff
@@ -1,7 +1,7 @@
 Fixes Issue: #<issue number>
 
 Here's the Pull Request Doc
-https://devtools-html.github.io/debugger.html/CONTRIBUTING.html#pull-requests
+https://github.com/devtools-html/debugger.html/blob/master/docs/pull-requests.md
```

Comment:
i think this will work
https://firefox-dev.tools/debugger.html/docs/pull-requests.html

Reply:
Oh, didn't see that! 

Updating now, thanks!